.github/workflows/release-please.yml

```diff
@@ -0,0 +1,69 @@
+name: Run Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-package:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write # Needed if using OIDC to get release secrets.
+      contents: write # Contents and pull-requests are for release-please to make releases.
+      pull-requests: write
+    outputs:
+      release-created: ${{ steps.release.outputs.release_created }}
+      upload-tag-name: ${{ steps.release.outputs.tag_name }}
+      gem-hash: ${{ steps.publish.outputs.gem-hash}}
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
Do you want v3?

Reply:
No, I'd rather keep all ruby SDKs on the same setup. Then I will change them all at once. I find that easier than doing them bit by bit.